### PR TITLE
LPS-168337 [Questions] Add ActivityRow and Components

### PR DESCRIPTION
**Pull request by kevenleone**

### CODE REVIEW

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -379,3 +379,60 @@ div {
...
+	color: #fff;
+	display: flex;
+	flex-direction: row;
+	font-size: 12px;
+	font-weight: bold;
+	height: 24px;
+	padding: 3px 8px;
+	width: 3rem;
+}
+.list-badges {
```
Comment (kevenleone) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss:396]:
```suggestion

.list-badges {
```

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/ActivityQuestionRow.es.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+/* eslint-disable default-case */
```
Comment (kevenleone) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/ActivityQuestionRow.es.js:1]: Remove

```suggestion

```

**Comment by juliocezar96** [comment]
Just starting reviewing :)

**Comment by brenobcos** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
resent: https://github.com/pt-liferay-solutions/liferay-portal/pull/2832

**Pull request closed without merging**